tests/test_connector.py

```diff
@@ -62,6 +62,17 @@ def send_my_event(self, event):
         with self.assertRaises(TypeError):
             MyConnector()
 
+    def test_event_subclasses(self):
+        class MyEvent(Message):
+            pass
+
+        class MyConnector(Connector):
+            @register_event(Message, include_subclasses=True)
+            def send_my_event(self, event):
+                pass
+
+        MyConnector({})
```

Comment:
Without trying it, I think you can test the behaviour here with this:
```suggestion
        c = MyConnector({})
        assert MyEvent in MyConnector.events
```

Reply:
Even neater!